+Base/uibuildmodule.m

```diff
@@ -0,0 +1,188 @@
+function [code,f] = uibuildmodule(varargin)
```

Comment:
This is actually in here by accident, while it is anticipated in the comments in `uigetcode` when called from PathManager, it is not ready to be used and not integrated to CC yet. I'll try and finish it before everything else is reviewed (it won't change how the other files are reviewed).